projects/vdk-core/src/vdk/internal/cli_entry.py

```diff
@@ -110,7 +111,6 @@ def vdk_main(
         """
         Main method of the CLI. It call all configuration and initialization hooks and start CLI
         """
-
         plugin_registry.add_hook_specs(CoreHookSpecs)
         plugin_registry.load_plugin_with_hooks_impl(builtin_hook_impl, "core-plugin")
 
```

Comment:
Could you not exit on this line if root_command is non instead of doing the check twice later on?

Reply:
We need to run all the default hooks as per - https://github.com/vmware/versatile-data-kit/pull/1241#discussion_r1004526029 , initialize configuration and logging.